pxtlib/service.ts

```diff
@@ -634,6 +634,29 @@ namespace ts.pxtc {
         }
     }
 
+
+    export function tsSnippetToPySnippet(param: string): string {
+        const keywords: pxt.Map<string> = {
+            "true": "True",
+            "false": "False",
+            "null": "None"
+        }
+        const key = keywords[param];
+        if (key) {
+            return key
+        }
+        if (param.includes(".")) {
```

Comment:
iirc the symbol info should have an array for all of the parameters and the entry should have an "isEnum" propeerty that you could check. This is probably fine, but it could get a little weird in some cases (e.g. if the default is a string with a period in it or something like ``assets.tilemap`whatever` ``)

Reply:
! okay I made a change to use the symbol info if it's available to check for enum. it'll be more accurate when dragging out the snippet than in the toolbox, but I think that's okay?